src/state/api.js

```diff
@@ -2,21 +2,21 @@ import axios from "axios"
 
 export const sendRequestAdduser = async (user) => {
 	try {
-		/* eslint-disable */
-		const response = await axios({
+		const request = {
+			/* eslint-disable */
 			method:"post",
 			url:"http://localhost:8082/api/users",
```

Comment:
create an `.env` file in the root directory and move `http://localhost:8082` to there. It should be something like `INTEGRATION-URL=http://localhost:8082`. you can use `https://github.com/andrewmclagan/react-env` to read environmental entries.